### PR TITLE
Update to 1.17.0

### PR DESCRIPTION
**Pull request by brianjmurrell**
Update packaging/.

Disable build on CentOS 7.

Sync with Fedora specfile.

### CODE REVIEW

**Review by JohnMalmberg — APPROVED** [review]

File: packaging/Dockerfile.mockbuild [hunk 1]
```diff
@@ -1,47 +1,74 @@
...
 #
 # 'recipe' for Docker to build an RPM
 #
 
 # Pull base image
-FROM fedora:latest
+ARG FVERSION=latest
+FROM fedora:$FVERSION
+# Needed for later use of FVERSION
+ARG FVERSION
```
Comment (jzarzycki) [inline packaging/Dockerfile.mockbuild:11]: What is the purpose of ```ARG FVERSION``` when there is ```ARG FVERSION=latest``` above?

**Review by jzarzycki — APPROVED** [review]

File: packaging/Dockerfile.mockbuild (hunk 1 above)
Comment (JohnMalmberg) [reply packaging/Dockerfile.mockbuild:11]: It is a documented feature of Dockerfiles.   The first ARG statements with default values above the FROM statement are only visible to the FROM statement.   The ARG statement with the same name, makes that ARG value visible to the rest of the Dockerfile.

**Review by JohnMalmberg — APPROVED** [review]

**Comment by brianjmurrell** [comment]
@jgmoore-or Does bumping this up to 1.17.0 seem a reasonable thing to do?  As we discussed previously, this repo is really only used by allowing us to build debuggable versions of UCX, but keeping it build-able (which it is not currently due to upstream having removed the release it was based on) seems like a good thing.  I'm just hoping 1.17.0 is a reasonable version for us to be doing that at currently.

**Review by JohnMalmberg — APPROVED** [review]

**Comment by jgmoore-or** [comment]
@brianjmurrell - I would like to get this done (upgrade to UCX 1.17). Once complete, it will be easy to add patches to the code for logging and debug, which will be useful at customer sites and potentially for CI runs.

**Review by jgmoore-or — APPROVED** [review]

**Pull request merged**